presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkBatchStreamReaders.java

```diff
@@ -280,21 +280,22 @@ OrcBatchRecordReader createRecordReader()
     {
         @SuppressWarnings("unused")
         @Param({
-                "boolean",
-
-                "tinyint",
-                "integer",
-                "bigint",
-                "decimal(10,5)",
-                "decimal(30,10)",
-
-                "timestamp",
-
-                "real",
-                "double",
-
-                "varchar",
-                "varbinary",
+//                "boolean",
```

Comment:
you will need to uncomment these

Reply:
Sorry, forgot to push the change to this